bela/README.md

```diff
@@ -0,0 +1,61 @@
+# Chirp + Bela
+
+Use the high audio fidelity Bela cape for BeagleBone Black board to send and
+receive data via sound with Chirp SDK !
+
+## Prerequisites
+
+- Follow the [Getting started with Bela](https://github.com/BelaPlatform/Bela/wiki/Getting-started-with-Bela) to get your board setup if it is not yet.
+- Sign up to Chirp at the [Chirp developer hub](https://developers.chirp.io) to download our SDK for free.
+- A microphone, the [Electret Microphone](https://www.sparkfun.com/products/12758) works well.
+- A speaker.
+
+## Setup
+
+You can connect the microphone the following way : 
+- GND --> P9_01
+- VCC --> P9_03
+- AUD --> LEFT IN - ADC1
+
+Use the connector shipped with Bela to connect a speaker to the LEFT OUT connector.
+
+This is an example of how the montage should look like :
```

Comment:
Is montage the right word here?

Reply:
I couldn't find any other word X) Any idea ?

Reply:
I thought a montage was a selection of pictures or footage cut together with some epic music. Probably something more boring like, see the image below for an example set up